libraries/api-extractor-model/package.json

```diff
@@ -5,7 +5,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/microsoft/rushstack.git",
-    "directory": "apps/api-extractor-model"
+    "directory": "libraries/api-extractor-model"
```

Comment:
We'll need a corresponding fix in https://github.com/microsoft/api-extractor.com-website/

Reply:
https://github.com/microsoft/api-extractor.com-website/pull/42/